indico/modules/events/abstracts/templates/display/submit_abstract.html

```diff
@@ -0,0 +1,28 @@
+{% from 'events/display/indico/_common.html' import render_users %}
+{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_fieldset %}
+
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title -%}
+    {% trans %}Submit abstract{% endtrans %}
+{%- endblock %}
+
+{%- block content %}
+    {% if cfa.is_open %}
```

Comment:
So basically what you're saying is that I would `return WPSubmitAbstract.render_template(...` twice, once `if not self.event_new.cfa.can_submit_abstracts(session.user):` and once at the very end...